### PR TITLE
feat: support 2FA otp

### PR DESCRIPTION
**Pull request by sgalcheung**
Implement [issue 142](https://github.com/florian-lefebvre/astro-integration-template/issues/142).

### CODE REVIEW

File: scripts/release.mjs [hunk 1]
```diff
@@ -35,12 +36,26 @@ const run = async (command, ...args) => {
...
 const main = async () => {
+	const config = {
+		options: {
+		otp: { type: "string", short: "o" },
+		OTP: { type: "string" },
+		},
+	};
+
+	const { values } = parseArgs(config);
+	const otpValue = values.otp || values.OTP;
```
Comment (florian-lefebvre) [inline scripts/release.mjs:47]:
```suggestion
	const { values } = parseArgs({
		options: {
			otp: { type: "string" }
		}
	});
```

File: scripts/release.mjs [hunk 2]
```diff
@@ -35,12 +36,26 @@ const run = async (command, ...args) => {
...
+	
 	await run("pnpm changeset version");
 	await run("git add .");
 	await run('git commit -m "chore: update version"');
 	await run("git push");
 	await run("pnpm --filter package-name build");
-	await run("pnpm changeset publish");
+	if (otpValue) {
```
Comment (florian-lefebvre) [inline scripts/release.mjs:54]:
```suggestion
	if (values.otp) {
```

**Comment by florian-lefebvre** [comment]
I wonder where we should document it, maybe in `DELETE_ME`

File: scripts/release.mjs [hunk 3]
```diff
@@ -35,12 +36,26 @@ const run = async (command, ...args) => {
...
+		await run(`pnpm changeset publish --otp=${otpValue}`);
```
Comment (sgalcheung) [inline scripts/release.mjs:55]:
```suggestion
		await run(`pnpm changeset publish --otp=${values.otp}`);
```

**Comment by sgalcheung** [comment]
I want to ignore the case of the input environment variable opt, so I defined lowercase and uppercase.
Whether we have a better way to handle this? Or using a standard way is acceptable.

**Comment by florian-lefebvre** [comment]
I think having it lowercase is better since it matches the changeset api. I don't think we need to allow a lot of things, it's a template. If people prefer another way, they can update the script after cloning it

**Review by Fryuni — APPROVED** [review]
Besides Florian's points LGTM.

Also think it might be worth documenting somewhere

**Pull request merged**